lms/static/js/groups/views/cohort_editor.js

```diff
@@ -5,10 +5,24 @@ var edx = edx || {};
 
     edx.groups = edx.groups || {};
 
+    edx.groups.reloadPage = function() {
+        location.reload(true);
+    };
+
+    edx.groups.setActionButtonsVisibility = function(showSettings) {
```

Comment:
This should within the Backbone view containing cohort-management-details, not as a function declared outside of the view class.